sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporterTest.java

```diff
@@ -0,0 +1,155 @@
+package com.daimler.sechub.sereco.importer;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.daimler.sechub.sereco.ImportParameter;
+import com.daimler.sechub.sereco.metadata.SerecoCodeCallStackElement;
+import com.daimler.sechub.sereco.metadata.SerecoMetaData;
+import com.daimler.sechub.sereco.metadata.SerecoSeverity;
+import com.daimler.sechub.sereco.metadata.SerecoVulnerability;
+import com.daimler.sechub.sereco.test.SerecoTestFileSupport;
+
+public class SarifV1JSONImporterTest {
+
+    private SarifV1JSONImporter importerToTest;
+    private String sarifBrakeman;
+    private String sarifEmptyResult;
+    private String sarifThreadflowsExample;
+    
+    @Before
+    public void before() {
+        importerToTest = new SarifV1JSONImporter();
+        sarifBrakeman = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_brakeman.json");
+        sarifEmptyResult = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_empty_results.json");
+        sarifThreadflowsExample = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_threadflows_example.json");
+    }
+
+    @Test
+    public void sarifreport_can_be_imported() {
+        /* prepare */
+
+        ImportParameter paramBrakeman = ImportParameter.builder().importData(sarifBrakeman).importId("id1").productId("SARIF").build();
+        ImportParameter paramThreadFlows = ImportParameter.builder().importData(sarifThreadflowsExample).importId("id1").productId("SARIF").build();
+
+        /* execute */
+        ProductImportAbility ableToImportBrakemanSarif = importerToTest.isAbleToImportForProduct(paramBrakeman);
+        ProductImportAbility ableToImportThreadFlowSarif = importerToTest.isAbleToImportForProduct(paramThreadFlows);
+
+        /* test */
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportBrakemanSarif);
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportThreadFlowSarif);
+    }
+
+    @Test
+    public void empty_sarifreport_throws_exception() {
+        
+        /* prepare */
+        /* test */
```

Comment:
This should be `/* execute + test */` the `assertThrows` executes `importResult` and asserts that the `IOException` is thrown.